components/SettingsComponent.vue

```diff
@@ -53,6 +53,9 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
   margin-top: 18px;
   padding-top: 18px;
 }
+.settingsList .rightAlign {
+  text-align: right;
+}
```

Comment:
I'm personally not a big fan of these kind of single-use classes.
I'd rather - and coincidentally already did so in the `feature/config-rewrite` branch - have the rule say `.settingsList th` and then drop the class attribute in the appropriate `<th>` tags.

Reply:
For these Vue SFC pages, I specifically tried to avoid any use of style arguments (although a few leaked in), because the SFC does a nice job of separating the logic, formatting, and data output.  So it's easy to look at the style section to see all formatting differences between the base app in a single place.

I still think this is correct (especially since this does abstract the formatting out into a single variable, if we ever want to change it in the future - for things in addition to just the text alignment), but I'm not going to die on this hill.

Reply:
What exactly do you consider correct, though? The `.rightAlign` class? Consider changing the text-alignment to be centered: You'd have to create a new class named `.centerAlign` _and_ change each and every place where the text-alignment is set currently. Having `.settingsList th` allows you to change the alignment in one singular line change.

Reply:
Sorry, you are right, I wasn't reading this correctly.  Will fix.